data/json/monsters/fish.json

```
@@ -1233,5 +1233,176 @@
+    "volume": "3000 ml",
+    "weight": "1600 g",
+    "reproduction": { "baby_egg": "egg_mackerel", "baby_count": 3, "baby_timer": 20 }
+  },
+  {
+    "id": "mon_fish_flounder",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_medium",
+    "name": { "str_sp": "flounder" },
+    "description": "A flatfish, the flounder, also known as a fluke, likes to stay camouflaged against the ocean floor.",
+    "volume": "2500 ml",
+    "weight": "1400 g",
+    "reproduction": { "baby_egg": "egg_flounder", "baby_count": 2, "baby_timer": 22 }
+  },
+  {
+    "id": "mon_fish_blackfish",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_small",
+    "name": { "str_sp": "black sea bass" },
+    "description": "A small, robust fish, the black sea bass is known for its dark, mottled color and meaty texture.",
+    "volume": "1500 ml",
+    "weight": "600 g",
+    "reproduction": { "baby_egg": "egg_blackfish", "baby_count": 2, "baby_timer": 15 }
+  },
+  {
+    "id": "mon_fish_porgy",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_small",
+    "name": { "str_sp": "scup" },
+    "description": "Also known as porgy, the scup is a small fish commonly found in Atlantic coastal waters, recognizable by its silver body with blue and purple tinges.",
+    "volume": "1300 ml",
+    "weight": "550 g",
+    "reproduction": { "baby_egg": "egg_porgy", "baby_count": 3, "baby_timer": 16 }
+  },
+  {
+    "id": "mon_fish_bluefish",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_small",
+    "name": { "str_sp": "bluefish" },
+    "description": "The bluefish, a moderately proportioned marine fish, is known for its aggressive nature and sharp teeth.  It sports a distinct blue-green coloration.",
+    "volume": "1600 ml",
+    "weight": "700 g",
+    "reproduction": { "baby_egg": "egg_bluefish", "baby_count": 2, "baby_timer": 18 }
+  },
+  {
+    "id": "mon_fish_cod",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_large",
+    "name": { "str_sp": "Atlantic cod" },
+    "description": "A large, white-meat fish known for its mild flavor and flaky texture.  Highly prized in both commercial and recreational fishing.",
+    "volume": "35000 ml",
+    "weight": "22 kg",
+    "reproduction": { "baby_egg": "egg_cod", "baby_count": 3, "baby_timer": 19 }
+  },
+  {
+    "id": "mon_fish_haddock",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_large",
+    "name": { "str_sp": "haddock" },
+    "description": "A popular food fish, the haddock is recognized by its dark lateral line and a black blotch above the pectoral fin.",
+    "volume": "25500 ml",
+    "weight": "11 kg",
+    "reproduction": { "baby_egg": "egg_haddock", "baby_count": 3, "baby_timer": 18 }
+  },
+  {
+    "id": "mon_fish_bluefin_tuna",
+    "type": "MONSTER",
+    "copy-from": "mon_fish_huge",
+    "name": { "str_sp": "Atlantic bluefin tuna" },
+    "description": "One of the largest and fastest fish in the ocean, the bluefin tuna is a streamlined predator with a dark blue back and silvery sides.",
```

```suggestion
    "description": "One of the largest and fastest fish in the waters of New England, the bluefin tuna is a streamlined predator with a dark blue back and silvery sides.",
```
Maybe that. One of the largest in the ocean is giving it way too much credit lmao.